static/js/TextsPropertiesMenu.jsx

```diff
@@ -109,26 +120,26 @@ function TextsPropertiesMenu(props) {
 
     const menuImg = (Sefaria.interfaceLang === 'hebrew') ? 'alef' : 'a';
     const button = (
-        <img src={`/static/icons/${menuImg}.svg`} />
+        <img src={`/static/icons/${menuImg}.svg`} alt="Text properties menu" />
     );
 
     return <PopoverMenu button={button} menu={menu} />;
 }
 
-TextsPropertiesMenu.proptypes = {
+TextsPropertiesMenu.propTypes = {
     sourceDir: PropTypes.string.isRequired,
     translationDir: PropTypes.string.isRequired,
-    showSource: PropTypes.bool,
-    showTranslation: PropTypes.bool,
-    layoutPreferences: PropTypes.object,
+    initialShowSource: PropTypes.bool,
```

Comment:
the typical naming convention is to change showSource in useState to showSourceState as opposed to initialShowSource as that sounds like the variable you are providing will change its data in the future